nvi-commons/src/test/java/no/sikt/nva/nvi/common/service/CandidateTest.java

```diff
@@ -97,23 +98,35 @@ class CandidateTest extends LocalDynamoTest {
     private static final String BASE_PATH = ENVIRONMENT.readEnv("CUSTOM_DOMAIN_BASE_PATH");
     private static final String API_DOMAIN = ENVIRONMENT.readEnv("API_HOST");
     public static final URI CONTEXT_URI = UriWrapper.fromHost(API_DOMAIN).addChild(BASE_PATH, "context").getUri();
+    private static final URI HARDCODED_INSTITUTION_ID = URI.create(
+        "https://example"
+        + ".org/hardCodedInstitutionId");
     private CandidateRepository candidateRepository;
     private PeriodRepository periodRepository;
 
     public static Stream<Arguments> candidateResetCauseProvider() {
+        //TODO: Test institution changed
```

Comment:
Added a TODO there when i noticed bug, created jira task: [Remove approval for institution A when creator affiliation changes to institution B](https://sikt.atlassian.net/browse/NP-48041)